### PR TITLE
Refactoring Siddhi Script JS extension 

### PR DESCRIPTION
**Pull request by AnuGayan**

### CODE REVIEW

File: component/pom.xml [hunk 1]
```diff
@@ -22,23 +22,22 @@
     <parent>
         <groupId>org.wso2.extension.siddhi.evalscript.js</groupId>
         <artifactId>siddhi-evalscript-js-parent</artifactId>
```
Comment (tishan89) [inline component/pom.xml]: change artifact id according to the convention

File: component/pom.xml [hunk 2]
```diff
@@ -22,23 +22,22 @@
...
-        <version>3.1.3-SNAPSHOT</version>
+        <version>4.0.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
     <artifactId>siddhi-evalscript-js</artifactId>
```
Comment (tishan89) [inline component/pom.xml]: change artifact id according to the convention

**Review by tishan89 — CHANGES_REQUESTED** [review]

File: component/pom.xml (hunk 2 above)
Comment (AnuGayan) [reply component/pom.xml]: @tishan89  Done

File: component/pom.xml (hunk 1 above)
Comment (AnuGayan) [reply component/pom.xml]: @tishan89  Done

File: component/pom.xml (hunk 2 above)
Comment (tishan89) [reply component/pom.xml]: You have missed to change here

Comment (AnuGayan) [reply component/pom.xml]: @tishan89  Sorry, now it's done

**Review by tishan89 — APPROVED** [review]

**Comment by tishan89** [comment]
approved. Will merge when Siddhi M9 is released

**Pull request merged**